app/views/custom_devise_mailer/proposed_org_approved.erb

```diff
@@ -0,0 +1,2 @@
+Your request to include <%= @org.name%> on the Harrow Community Network has been granted.
+<%= link_to "Take over your organisation on HCN by clicking here.", accept_invitation_url(@resource, :invitation_token => @resource.raw_invitation_token)%>
```

Comment:
we'll need to check with the client, but I think "Get admin access" might be better than "take over"